pepys_import/core/store/common_db.py

```diff
@@ -204,13 +216,14 @@ def commit(self, data_store, change_id):
         # Since measurements are saved by their importer names, iterate over each key
         # and save its measurement objects.
         extraction_log = list()
-        for key in self.measurements.keys():
-            print(f"Submitting measurements extracted by {key}.")
-            for file in tqdm(self.measurements[key]):
-                file.submit(data_store, change_id)
-            extraction_log.append(
-                f"{len(self.measurements[key])} measurements extracted by {key}."
-            )
+        for key in self.measurements:
```

Comment:
I know it's a `key` in computing terms,  but in business domain it's a `parser` isn't it?

Reply:
Yes, it is. I'll change it now

Reply:
Corrected:
https://github.com/debrief/pepys-import/pull/244/commits/70a71543afae432dd84ec81316cfe7421b7c03e3